.github/workflows/scorecards.yml

```diff
@@ -19,6 +19,8 @@ jobs:
       security-events: write
       actions: read
       contents: read
+      # Needed to publish results
+      id-token: write
```

Comment:
Where are these results published?  Where is it writing too?

Reply:
Ah, sorry, I thought you already knew.

The Scorecard Action publishes results to a public dataset, which can be explored via BigQuery or through https://securityscorecards.dev/viewer/?uri=github.com/emscripten-core/emscripten.

If you'd rather the Action not publish this information, I can instead remove the `publish-results: true` argument passed to the Action.

Reply:
Note that the information is public and Scorecard also runs a weekly scan of ~1M projects (including emscripten), so publishing results simply defines whether the results will be up-to-date or a few days behind. Also, since you're running Scorecard with a PAT, your results are actually more complete than those from the cronjob, which doesn't have access to some relevant information.

Reply:
I don't mind the information being public, but I'm still a little confused.  Is the information being published it the emscripten github org somewhere?   Does this line give it permission to write back to our github project?  Or is it writing to securityscorecards.dev?

i.e can this action do stuff like push changes to close bugs?

Reply:
Oh no, `id-token: write` simply gives the workflow access to a token it can use to identify itself (it's an OIDC token). This doesn't give it permission or access to _**do**_ anything within the org or repo (that's controlled by all the other permission types like `contents: write`, etc). It simply enables the workflow to say "hey, I'm a workflow from emscripten-core/emscripten, you can trust me when I say this is the project's Scorecard results".

Reply:
sgtm